scripts/leetcode.js

```diff
@@ -298,6 +298,13 @@ function checkElem(elem) {
   return elem && elem.length > 0;
 }
 
+/* Convert html to markdown */
+import html2md from './h2m'
```

Comment:
have u experimented it on the chrome extension? i'm not familiar with the usage of libraries within chrome API

Reply:
sorry, I am not familiar with chrome extension.

This PR's change doesn't work on chrome extension.

I implemented this function in another branch in a better way, and tested it with my local chrome extension.

Reply:
right, i was referring to your local chrome extension. could u explain why it's better than the current approach, i.e. html parsing?

Reply:
Actually，I write leetcode problem notes based on a leet hub repository.
I made a turndown-based-converter that convert html text to md text.It actually does something I expected.
I think there may be a lot people would like to write their thinking about problem in leethub-generated file.

Reply:
Add:
A pure md file is more editable than normal html file.
If users want to write here than to read here,md file is a better choice.